lib/modules/dosomething/dosomething_reportback/includes/Reportback.php

```diff
@@ -121,12 +121,22 @@ public static function find(array $filters = []) {
       throw new Exception('No reportback data found.');
     }
 
-    foreach($results as $item) {
-      // @TODO: remove need for passing variable for constructor check.
-      $reportback = new static(['ignore' => TRUE]);
-      $reportback->build($item);
+    if (isset($filters['load_user_data'])) {
```

Comment:
the `dosomething_helpers_isset()` function is just a helper function to help deal w/ the whole `isset()` crap :)